engine/src/main/java/org/hibernate/validator/constraints/SafeHtml.java

```diff
@@ -65,6 +65,12 @@
 	Tag[] additionalTagsWithAttributes() default { };
 
 	/**
+	 * @return Base URI to be used in resolving relative links to absolute ones.
```

Comment:
Should there be a note here stating that relative URLs will fail validation if baseUri is not set?